libbz2-rs-sys/src/huffman.rs

```diff
@@ -155,17 +155,12 @@ pub(crate) fn make_code_lengths(len: &mut [u8], freq: &[i32], alphaSize: usize,
     }
 }
 
-pub(crate) fn assign_codes(
-    code: &mut [i32],
-    length: &[u8],
-    minLen: i32,
-    maxLen: i32,
-    alphaSize: usize,
-) {
+#[inline(always)]
```

Comment:
Does this really have to be `#[inline(always)]` rather than `#[inline]`? `#[inline(always)]` makes it slower to do a debug build.

Reply:
hmm i believe that it did matter earlier, but now it doesn't. I'll remove it.